dist/tools/suit/Makefile

```
@@ -0,0 +1,10 @@
+PKG_NAME=suit-manifest-generator
+PKG_URL=https://github.com/ARMmbed/suit-manifest-generator.git
+PKG_VERSION=959dd3fca00806faf06699c449653cd48b84841e
+PKG_LICENSE=Apache2.0
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all:
+	$(Q)rm -fr suit-manifest-generator
+	-$(Q)ln -s $(PKGDIRBASE)/suit-manifest-generator .
```

This looks ugly. You could either change the value of `SUIT_TOOL` defined in

https://github.com/RIOT-OS/RIOT/blob/453d0478dce107f810f4690093899e36faaf7cb8/makefiles/suit.base.inc.mk#L2-L3

to `$(PKGDIRBASE)/suit-manifest-generator/bin/suit-tool`

Or specify `PKG_SOURCE_DIR` before the include of `pkg.mk`:

```mk
PKG_SOURCE_DIR ?= $(CURDIR)
include $(RIOTBASE)/pkg/pkg.mk
```
Both options are untested but I think they are better than the current symlink approach.

In any case:
```suggestion
	:
```